src/pages/messages/components/TicketListItem.tsx

```diff
@@ -66,11 +67,8 @@ export const TicketListItem = ({ ticket }: TicketListItemProps) => {
   const isOnMyPageMessages = window.location.pathname === UrlPathTemplate.MyPageMyMessages;
 
   return (
-    <SearchListItem
-      sx={{
-        p: 0,
-        bgcolor: !viewedByUser ? 'tertiary.light' : 'white',
-      }}>
+    <TaskListItem
+      sx={{ bgcolor: !viewedByUser ? 'tertiary.light' : 'white', borderLeftColor: getTicketColor(ticket.type) }}>
```

Comment:
Colored border to the left on an item in list of tickets. Different colors depending on type of ticket